application/models/Project_model.php

```diff
@@ -419,6 +493,7 @@ public function get_authors_by_project($project_id, $type= 'author')
         return false;
     }
 
+
```

Comment:
Extra whitespace?

Reply:
Not my forte, tracking these down! I've made an effort. Have possibly been able to stomp on this one now.